CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Changelog
 =========
 
+* **2016-02-10**: Avoid warning on empty token provider value and missing
+                  `allow_empty` key (#66)
```

Comment:
Actually we don't need this changelog entry do we? Its a bug fix for a new feature..